graphql/query_resolver.go

```diff
@@ -1024,6 +1024,27 @@ func (r *queryResolver) Version(ctx context.Context, versionID string) (*restMod
 	return &apiVersion, nil
 }
 
+// Image is the resolver for the image field returning information about an image including kernel, version, ami, name, and last deployed time.
+func (r *queryResolver) Image(ctx context.Context, imageID string) (*Image, error) {
+	config, err := evergreen.GetConfig(ctx)
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("getting evergreen configuration: '%s'", err.Error()))
+	}
+	c := thirdparty.NewRuntimeEnvironmentsClient(config.RuntimeEnvironments.BaseURL, config.RuntimeEnvironments.APIKey)
+	result, err := c.GetDistroInfo(ctx, imageID)
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("getting image info: '%s'", err.Error()))
+	}
+	image := Image{
+		Name:         result.Name,
+		VersionID:    result.VersionID,
+		Kernel:       result.Kernel,
+		LastDeployed: result.LastDeployed,
+		Ami:          result.AMI,
+	}
+	return &image, nil
+}
```

Comment:
ah okay so what you want to do to avoid this is to create a new entry in `gqlgen.yml` that maps the GQL object to the SVC object. Then you can regenerate the schema via `make gqlgen`, and you should be able to just return `result`